addons/mail/controllers/discuss.py

```diff
@@ -198,6 +198,13 @@ def mail_init_messaging(self, **kwargs):
             return guest.sudo()._init_messaging()
         raise NotFound()
 
+    @http.route('/mail/channel/channel_members', methods=['POST'], type='json', auth='public')
```

Comment:
Route could be `mail/channel/members` (`channel_members`, "channel" is redundant)